src/components/inputs/localized-text-input/localized-text-input.js

```diff
@@ -180,12 +183,20 @@ export default class LocalizedTextInput extends React.Component {
       hasErrorOnRemainingLanguages ||
       props.hideExpansionControls ||
       state.areLanguagesOpened;
-    return { areLanguagesOpened };
+
+    const id = do {
```

Comment:
Let's wait for #311 and use `withId` instead?

Reply:
I closed #311 as it wasn't feasible yet. Let's continue the way it is right now then.